packages/molly/src/routes/chat/components/chatInput.svelte

```diff
@@ -0,0 +1,95 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import sendChat from '../logo/paper-plane.svg';
+	export let img: string = sendChat;
+	let textarea: HTMLTextAreaElement;
+
+	const dispatch = createEventDispatcher();
+
+	interface Message {
```

Comment:
We could move this to `@mollify/types` as MollifyMessage, or use the types from the `OpenAI` package

Reply:
Either sounds good to me, not sure how you would like to do it!